code/modules/multiz/_HELPERS.dm

```diff
@@ -0,0 +1,243 @@
+/* Multi-Z code was ported from Polaris, which operates under the AGPL v3 license.
+Permission for its use was obtained on 11/12/2017 from Neerti in the Polaris Discord. */
+
+#define isopenspace(A) istype(A, /turf/simulated/open)
+#define isvisiblespace(A) is_type_in_list(A, list(/turf/simulated/open, /turf/simulated/floor/glass))
+#define OPENSPACE_PLANE_START -23
+#define OPENSPACE_PLANE_END -8
+#define OPENSPACE_PLANE -25
+#define OVER_OPENSPACE_PLANE -7
+//#define QDELETED(X) (!X || X.gc_destroyed)  Polaris has this
+
+/obj/effect/landmark/map_data
+	name = "Unknown"
+	desc = "An unknown location."
+	invisibility = 101
+
+	var/height = 1     ///< The number of Z-Levels in the map.
```

Comment:
number of Z levels can already be counted on the map itself just by counting the z level datums

Reply:
Moved to map datum